### PR TITLE
Add attention span on sending reset password email

### PR DESCRIPTION
**Pull request by albertmourato**
Closes https://github.com/wedeploy/wedeploy.com/issues/496

### CODE REVIEW

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 1]
```diff
@@ -363,6 +363,24 @@ curl -X "POST" "https://<serviceID>-<projectID>.wedeploy.io/user/recover" \
 
 </article>
 
+<aside>
+
+###### <span class="icon-16-alert"></span> Attention
+
+To send password reset email, you must have an [email service](/docs/email/getting-started/) with `email` as `id` in the same project. 
+If you don't want to use `email` as service id, you can add this environment variable to your `auth` service:
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: Let's add an extra line break here.

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 2]
```diff
@@ -363,6 +363,24 @@ curl -X "POST" "https://<serviceID>-<projectID>.wedeploy.io/user/recover" \
 
 </article>
 
+<aside>
+
+###### <span class="icon-16-alert"></span> Attention
+
+To send password reset email, you must have an [email service](/docs/email/getting-started/) with `email` as `id` in the same project.
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: "To send password reset emails,"
"as the `id`"

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 3]
```diff
@@ -363,6 +363,24 @@ curl -X "POST" "https://<serviceID>-<projectID>.wedeploy.io/user/recover" \
...
+<aside>
+
+###### <span class="icon-16-alert"></span> Attention
+
+To send password reset email, you must have an [email service](/docs/email/getting-started/) with `email` as `id` in the same project. 
+If you don't want to use `email` as service id, you can add this environment variable to your `auth` service: 
+```text/x-sh
+{
+  "id": "auth",
+  "image": "wedeploy/auth:2.0.0"
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: Let's use the dynamic image link: `@site.version.image.auth@`

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 4]
```diff
@@ -363,6 +363,24 @@ curl -X "POST" "https://<serviceID>-<projectID>.wedeploy.io/user/recover" \
...
+  "env": {
+    "WEDEPLOY_AUTH_EMAIL_SERVICE_URL": "myEmailService"
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: Let's make it even more clear: `"myEmailServiceId"`

**Comment by albertmourato** [comment]
Done! :)

**Pull request merged**